plugins/modules/aci_pim_route_map_entry.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_pim_route_map_entry
+short_description: Manage Protocol-Independent Multicast (PIM) Route Map Entry (pim:RouteMapEntry)
+description:
+- Manage PIM Route Map Entries for the PIM route Map Policies on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  pim_route_map_policy:
+    description:
+    - The name of the PIM Route Map policy.
+    type: str
+    aliases: [ route_map_policy_name ]
+  order:
+    description:
+    - The PIM Route Map Entry order.
+    type: int
+  source_ip:
+    description:
+    - The Multicast Source IP.
+    type: str
+  group_ip:
+    description:
+    - The Multicast group IP/prefix.
```

Comment:
Should we remove /Prefix and captitalize group to keep consistent with UI?
```suggestion
    - The Multicast Group IP.
```